From 1a0b991f7111cc5b6c69f21760f131f0509ce7f1 Mon Sep 17 00:00:00 2001
From: Chen Lei <lei.chen@rock-chips.com>
Date: Tue, 8 Jan 2019 16:18:31 +0800
Subject: [PATCH] phy/rockchip: phy-rockchip-emmc: Change EMMC calibration
 timeout to 500us

This modification prevents the emmc from issuing calibration timeout
errors at low temperatures

Change-Id: I5c3fddb8ea0ecf32d43e8e8190297638a1040a98
Signed-off-by: Chen Lei <lei.chen@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-emmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-emmc.c b/drivers/phy/rockchip/phy-rockchip-emmc.c
index b69a0a796473..6ee26753ea2a 100644
--- a/drivers/phy/rockchip/phy-rockchip-emmc.c
+++ b/drivers/phy/rockchip/phy-rockchip-emmc.c
@@ -178,7 +178,7 @@ static int rockchip_emmc_phy_power(struct phy *phy, bool on_off)
 	if (regmap_read_poll_timeout(rk_phy->reg_base,
 				     rk_phy->reg_offset + GRF_EMMCPHY_STATUS,
 				     caldone, PHYCTRL_IS_CALDONE(caldone),
-				     5, 50)) {
+				     5, 500)) {
 		pr_err("rockchip_emmc_phy_power: caldone timeout.\n");
 		return -ETIMEDOUT;
 	}
-- 
2.35.3

